src/site/content/en/fast/signed-exchanges-webpackager/index.md

```diff
@@ -0,0 +1,467 @@
+    named `Web Packager Demo` that has the [common
+    name](https://knowledge.digicert.com/solution/SO7239.html) `example.com`. The
+    common name should be the fully qualified domain name of the site that contains
+    the content that you want to package as SXG.
+
+    In a production SXG setup, this would be a site that you own. However, in a
+    testing environment like the one described in these instructions, it can be any
+    site.
+
+3. Create a certificate that has the `CanSignHttpExchanges` extension.
+
+    ```shell
+    openssl x509 -req -days 90 -in cert.csr -signkey priv.key -out cert.pem -extfile &lt;(echo -e "1.3.6.1.4.1.11129.2.1.22 = ASN1:NULL\nsubjectAltName=DNS:example.com")
+    ```
+
+    This command uses the private key and the CSR created in steps 1 and 2 to create the
+    certificate file `cert.pem`. The `-extfile` flag associates the certificate with
+    the `CanSignHttpExchanges` certificate extension (`1.3.6.1.4.1.11129.2.1.22` is
+    the [object
+    identifier](https://access.redhat.com/documentation/en-us/red_hat_certificate_system/9/html/administration_guide/standard_x.509_v3_certificate_extensions)
+    for the `CanSignHttpExchanges` extension). In addition, the `-extfile` flag also
+    defines `example.com` as a [Subject Alternative
+    Name](https://en.wikipedia.org/wiki/Subject_Alternative_Name). 
+
+    If you are curious about the contents of `cert.pem`, you can view them using the
+    following command:
+
+    ```shell
+    openssl x509 -in cert.pem -noout -text
+    ```
+
+    You are done creating private keys and certificates. You will need the
+    `priv.key` and `cert.pem` files in the next section.
+
+
+### Setup the Web Packager server for testing
+
+
+#### Prerequisites
+
+
+
+1. Install [Web Packager](https://github.com/google/webpackager).
+
+    ```shell
+    git clone https://github.com/google/webpackager.git
+    ```
+
+
+
+2. Build `webpkgserver`.
+
+    ```shell
+    cd webpackager/cmd/webpkgserver
+    go build .
+    ```
+
+    `webpkgserver` is a specific binary within the Web Packager project.
+
+
+
+3. Verify that `webpkgserver` has been installed correctly.
+
+    ```shell
+    webpkgserver --help
+    ```
+
+    This command should return information about the usage of `webpkgserver`. If
+    this does not work, a good first troubleshooting step is to verify that your
+    [GOPATH](https://golang.org/doc/gopath_code.html#GOPATH) is configured
+    correctly.
+
+
+#### Instructions
+
+
+
+1. Navigate to the `webpackager` directory (you might already be in this
+   directory).
+
+    ```shell
+    cd /path/to/cmd/webpkgserver
+    ```
+
+
+
+2. Create a `webpkgsever.toml` file by copying the example.
+
+    ```shell
+    cp ./webpkgserver.example.toml ./webpkgserver.toml
+    ```
+
+    This file contains the configuration options for `webpkgserver`.
+
+
+
+3. Open `webpkgserver.toml` with an editor of your choice and make the following
```

Comment:
You might want to mention this optional TOML config:  KeepNonSXGPreloads (default = false).
  # Include preload link headers that lack the corresponding allowed-alt-sxg
  # with a valid header-integrity.

The TOML describes what it's for.

Reply:
If you mention this option, you might want to disclaim that doing so could make the SXG ineligible for the Google SXG cache per [these requirements](https://github.com/google/webpackager/blob/master/docs/cache_requirements.md).